tests/utils/context.py

```diff
@@ -30,8 +31,9 @@
     # Path to the Python module
     sys.path.insert(0, '/usr/lib/kicad-nightly/lib/python3/dist-packages')
 import pcbnew
-m = re.search(r'(\d+)\.(\d+)\.(\d+)', pcbnew.GetBuildVersion())
 logging.debug(pcbnew.GetBuildVersion())
+m=re.search(r'(\d+)\.(\d+)\.(\d+)', pcbnew.GetBuildVersion())
```

Comment:
This should trigger a flake8 error because it lacks the spaces around `=`

Reply:
Probably should, don't know why they were removed.